packages/react-dom/src/__tests__/ReactDOMAttribute-test.js

```diff
@@ -126,5 +126,14 @@ describe('ReactDOM unknown attribute', () => {
 
       expect(el.firstChild.getAttribute('helloworld')).toBe('something');
     });
+
+    it('does not warn on camelCase unknown attributes with value of undefined', () => {
+      const el = document.createElement('div');
+
+      ReactDOM.render(<div helloWorld={undefined} />, el);
+
+      expect(el.firstChild.getAttribute('helloworld')).toBeNull();
+      expect(el.firstChild.getAttribute('helloWorld')).toBeNull();
```

Comment:
Quick question for context, why are you asserting it twice?

Reply:
Sorry about the late response @robarnold .  I'm actually not asserting the same thing twice, either here or in the other test.  Here for example, the "W" is capitalized in one but not the other.  

The reason for this is that if the value of the attribute is not undefined or null, the attribute will appear in the DOM, but lower-cased, not camel-cased as it is in the JSX.

![image](https://user-images.githubusercontent.com/34415120/63814372-8b08df00-c928-11e9-978a-bb9a9ce70734.png)


![image](https://user-images.githubusercontent.com/34415120/63814359-85ab9480-c928-11e9-9cdd-ac44744fadf2.png)

I didn't want the test to depend on this detail however.

I suppose my test doesn't need to bother with this, and could avoid this detail by just using an attribute name that consists of a single, lower-cased word. 

What do you think?

Reply:
@rodoabad ,  I made the changes I mentioned in my response so that  it won't be as confusing and consistent with the other tests.  I will resolve the conversation, please feel free to make more comments.  Thanks for the input.